platform/comm/src/main/java/com/fmicodes/comm/services/UnsplashService.java

```diff
@@ -0,0 +1,50 @@
+package com.fmicodes.comm.services;
+
+import org.asynchttpclient.AsyncHttpClient;
+import org.asynchttpclient.DefaultAsyncHttpClient;
+import org.asynchttpclient.Response;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.springframework.stereotype.Service;
+
+import java.util.concurrent.ExecutionException;
+
+@Service
+public class UnsplashService {
+
+    public String getUnsplashImage(String location) {
+        String destinationsResponse;
+        AsyncHttpClient client = new DefaultAsyncHttpClient();
+        try {
+            Response response = client.prepare("GET", "https://api.unsplash.com/search/photos?page=1&query=" + location + " landscape")
+                    .setHeader("Authorization", "Client-ID " + "7394sfnNucC9j1r5olgGsJ2-hiE6_28Xdi7R1KBC1rE")
```

Comment:
Isn't this id supposed to be private data?

Reply:
You're absolutely right - I will add it to the OS only "credentials.yaml" as separate value